package/gluon-respondd/src/respondd.c

```diff
@@ -211,11 +212,24 @@ static struct json_object * get_rootfs_usage(void) {
 	return jso;
 }
 
+static struct json_object * get_time(void) {
+	struct timespec now;
+
+	if (clock_gettime(CLOCK_REALTIME, &now) != 0)
+		return NULL;
+
+	return json_object_new_int64(now.tv_sec);
+}
+
 static struct json_object * respondd_provider_statistics(void) {
 	struct json_object *ret = json_object_new_object();
 
 	json_object_object_add(ret, "node_id", gluonutil_wrap_and_free_string(gluonutil_get_node_id()));
 
+	json_object *time = get_time();
+	if (time != NULL)
+		json_object_object_add(ret, "time", time);
```

Comment:
We could do without the NULL check, as *json_object_object_add* would just add a JSON *null* value. All data consumers need to be able to deal with broken data anyways, and it's not an error that should actually ever happen.